src/jltypes.c

```diff
@@ -3534,11 +3534,12 @@ void jl_init_types(void)
     jl_methtable_type->name->mt = jl_new_method_table(jl_methtable_type->name->name, jl_current_module);
     jl_methtable_type->super = jl_any_type;
     jl_methtable_type->parameters = jl_emptysvec;
-    jl_methtable_type->name->names = jl_svec(6, jl_symbol("name"), jl_symbol("defs"),
+    jl_methtable_type->name->names = jl_svec(8, jl_symbol("name"), jl_symbol("defs"),
                                              jl_symbol("cache"), jl_symbol("max_args"),
-                                             jl_symbol("kwsorter"), jl_symbol("module"));
-    jl_methtable_type->types = jl_svec(6, jl_sym_type, jl_any_type, jl_any_type, jl_any_type,
-                                       jl_any_type, jl_any_type);
+                                             jl_symbol("kwsorter"), jl_symbol("module"),
+                                             jl_symbol(""), jl_symbol(""));
+    jl_methtable_type->types = jl_svec(8, jl_sym_type, jl_any_type, jl_any_type, jl_any_type/*jl_long*/,
+                                       jl_any_type, jl_any_type/*module*/, jl_any_type/*long*/, jl_any_type/*int32*/);
```

Comment:
Debugging only? (Also note that there's also a `int ncalls;` in between for `JL_GF_PROFILE`)